### PR TITLE
Add a comment about the pen image

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
....

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
Wait for build to see if escaping works

**Comment by RonaldDijkstra** [comment]
Fine

File: config.rb [hunk 1]
```diff
@@ -322,7 +322,9 @@ def call(exception, locale, key, options)
   activate :asset_hash, ignore: [
     "images/logos/defacto.png",
     "images/blog/featured",
-    "images/blog/flexible-cover-images-using-intrinsic-ratio/aspect-ratio-demo.png"
+    # Image below is used in sn3p's codepen https://codepen.io/snap/pen/NGxgLr
+    "images/blog/flexible-cover-images-using-intrinsic-ratio/" \
```
Comment (sn3p) [inline config.rb]: Why? Line too long?
This looks horrible, I'd keep one line.

**Comment by RonaldDijkstra** [comment]
Yes line length 😅

**Pull request merged**